File: Modul_4/Soal_1/PRAK401-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main(void) {
    int jmlMurid = 50, angka;
    char simbol;

    scanf("%d %c", &angka, &simbol);
    for(int i = 1; i <= jmlMurid; i++) {
        if(i == angka || i % angka == 0) {
            printf("%c ", simbol);
        }
        else {
            printf("%d ", i);
        }
    }
}

File: Modul_4/Soal_1/PRAK401-2310817210015-MuhammadBukhariFitri.py
jmlMurid = 50

input = input().split()

if len(input) == 2:
    angka = int(input[0])
    simbol = input[1]

for i in range(1, jmlMurid + 1) :
    if(i == angka or i % angka == 0) :
        print(simbol, end=" ");
    else :
        print(i, end=" ");